docs/manual/manual_dronecan.rst

```diff
@@ -44,11 +44,274 @@ Servo modules do not support DroneCAN, as shown in the table below.
 	| Vertiq 2306 | .. centered:: |:x:|                |
 	+-------------+------------------------------------+
 
+.. _standard_dronecan_support:
+
 Standard DroneCAN Support
 ================================
-Vertiq modules that support DroneCAN support a standard set of messages, requests, and configuration parameters. For information on the DroneCAN messages, 
-requests, and parameters supported by Vertiq modules, see the `Standard DroneCAN Support documentation <https://www.vertiq.co/s/Preliminary-Standard-DroneCAN-UAVCANv0-Support.pdf>`_ 
-on Vertiq’s website.
+This section details the standard DroneCAN messages supported across all Vertiq DroneCAN modules currently. The structure and contents of these messages are defined by the 

```

Comment:
Matt mentioned on my initial pass of the Fortiq docs to maybe avoid words like "currently." If it changes, we'll just update the docs.